DashboardFrontend/ViewModels/PerformanceChart.cs

```diff
@@ -43,8 +36,9 @@ public PerformanceViewModel()
                     GeometryStroke = new SolidColorPaint(new SKColor(92, 84, 219)),
                     GeometrySize = 3,
                     TooltipLabelFormatter = e => Series?.ElementAt(0).Name + "\n" +
-                                                 DateTime.FromOADate(e.SecondaryValue).ToString("HH:mm:ss") + "\n" +
+                                                 DateTime.FromOADate(e.SecondaryValue).ToString("HH:mm:ss", new CultureInfo("en-US")) + "\n" +
```

Comment:
Hvorfor en-US? hvad blev der af da-DK
```suggestion
                                                 DateTime.FromOADate(e.SecondaryValue).ToString("HH:mm:ss", new CultureInfo("da-DK")) + "\n" +
```